sdk/core/azure-core/azure/core/rest/_helpers.py

```diff
@@ -138,8 +138,7 @@ def set_content_body(content: Any) -> Tuple[MutableMapping[str, str], Optional[C
         "We expect 'content' to either be str, bytes, a open file-like object or an iterable/asynciterable."
     )
 
-def set_json_body(json):
-    # type: (Any) -> Tuple[Dict[str, str], Any]
+def set_json_body(json: Any) -> Tuple[Dict[str, str], Any]:
```

Comment:
is `json` really `typing.Any`? Can't we be a bit more specific here?

Reply:
Since it was not the focus of the PR, I stupidly just converted the comments for now. I agree there is a huge typing work in Azure.Core, I was planning that in a second PR.